.github/workflows/push.yaml

```diff
@@ -3,6 +3,7 @@ name: ci
 on:
   push:
     branches: main
+  workflow_dispatch: {}
```

Comment:
look at that magic... Can you comment why this unreadable blob is here which will also help have it not removed in the future? :D

Reply:
done